packages/framer-motion/package.json

```diff
@@ -39,7 +39,7 @@
         "clean": "rm -rf types dist lib",
         "test": "yarn test-server && yarn test-client",
         "test-ci": "yarn test",
-        "test-client": "jest --config jest.config.json --max-workers=2",
+        "test-client": "jest --config jest.config.json --max-workers=2 src/animation/__tests__/css-variables.test.tsx",
```

Comment:
Was this for debugging? should we remove it?

Reply:
thanks good catch